bs/src/tablecloth.ml

```diff
@@ -36,7 +169,7 @@ module List = struct
 
   let elemIndex ~(value : 'a) (l : 'a list) : int option =
     l
-    |> Array.of_list
+    |> Caml.Array.of_list
     |> Js.Array.findIndex (( = ) value)
```

Comment:
Belt.List.toArray

Reply:
Why the preference of belt over the standard library?

Reply:
Because presumably the Belt ones are more optimized (and similarly, I expect the Jane Street ones are more optimized on the native side of things).